.github/release.yml

```diff
@@ -0,0 +1,34 @@
+changelog:
+  categories:
+    - title: 🚨 Breaking Changes
+      description: Changes that break existing functionality
+      labels:
+        - breaking
+    - title: ✨ New Features
+      description: New features and enhancements
+      labels:
+        - feature
```

Comment:
This is the mapping of the labeler: https://github.com/action-runner/conventional-labeler/blob/2c9e06155ac5585f82ca9440b3e5edba9e81d433/src/client/conventional_commit.ts#L4-L16

```
    feat: "enhancement",
    fix: "bug",
    docs: "documentation",
    style: "style",
    refactor: "refactor",
    perf: "performance",
    test: "test",
    chore: "chore",
    build: "build",
```

IIUC there won't be any label "feature" but we should match "enhancement" 

can you please confirm?

Reply:
Should be good now